scripts/winesapos-install.sh

```diff
@@ -809,7 +809,39 @@ elif [[ "${WINESAPOS_DE}" == "plasma" ]]; then
         pacman_install_chroot manjaro-kde-settings manjaro-settings-manager-knotifier
         # Install Manjaro specific KDE Plasma theme packages.
         pacman_install_chroot plasma6-themes-breath plasma6-themes-breath-extra breath-wallpapers sddm-breath-theme
-    fi
+
+elif [[${WINESAPOS_BOOTLOADER} == grub]]; then
+   pacman_install_chroot grub os-prober dosfstools xz bash gettext
+   sudo grub-install ${WINESAPOS_DEVICE}
+   GRUB_CFG_PATH="/boot/grub/grub.cfg"
+   touch $GRUB_CFG_PATH
+   echo "set default=0" >> $GRUB_CFG_PATH
+   echo "set timeout=5" >> $GRUB_CFG_PATH
+   echo " " >> $GRUB_CFG_PATH
+   echo "menuentry "winesapOS" {" >> $GRUB_CFG_PATH
+   echo "    set root='hd0,msdos1'" >> $GRUB_CFG_PATH
+   echo "linux /boot/vmlinuz-linux-lts root=/dev/sda1 rw quiet" >> $GRUB_CFG_PATH
+   echo "initrd /boot/initramfs-linux.img" >> $GRUB_CFG_PATH
+   echo "}" >> $GRUB_CFG_PATH
+   echo " " >> $GRUB_CFG_PATH
+   echo "menuentry "winesapOS (Fallback)" {" >> $GRUB_CFG_PATH
+   echo "    set root='hd0,msdos1'" >> $GRUB_CFG_PATH
+   echo "    linux /boot/vmlinuz-linux root=/dev/sda1 rw" >> $GRUB_CFG_PATH
+   echo "    initrd /boot/initramfs-linux-fallback.img" >> $GRUB_CFG_PATH
+   echo "}" >> $GRUB_CFG_PATH
+   sudo grub-mkconfig -o /boot/grub/grub.cfg
+elif [[ "${WINESAPOS_BOOTLOADER}" == systemd ]]; then
+      pacman_install_chroot systemd
+      touch /boot/loader/loader.conf
+      echo "default arch
+      timeout 4
+      console-mode max" >> /boot/loader/loader.conf
+      
+      touch /boot/loader/entries/arch.conf
+      echo "title   winesapOS
+      linux   /vmlinuz-linux
+      initrd  /initramfs-linux.img
+      options root=PARTUUID=winesapOS rw" >> /boot/loader/entries/arch.conf
```

Comment:
I don't think this will work. A UUID is supposed to be randomly generated 16 character string. `root=LABEL=winesapos-root` might work.

Could you do a build with the environment variable `WINESAPOS_BOOTLOADER=systemd-boot` and see if it works? I've never used systemd-boot before so I don't know what to expect.

Reply:
It is likely that it won't work unless the `UUID` is correct. We might be able to resolve this by using the output from the `blkid -s UUID` command.

Reply:
We might be able to resolve this by using the output from the blkid -s UUID command."

Reply:
We might be able to resolve this by using the output from the blkid -s UUID command.